index.html

```diff
@@ -232,24 +232,33 @@ <h2 id="skills" class="skills"><img src="assets/img/Rectangle56.png" alt="">Skil
       </section>
 
       <section id="contact">
-        <h2 id="contact-description">I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.</h2>
-        <form action="https://formspree.io/f/moqrnlgd" method="post" id="form">
-          <input type="text" name="name" id="contact-name" placeholder="Full name" maxlength="30" required>
-          <input type="email" name="email" id="contact-email" placeholder="Email address" required>
-          <textarea name="message" id="contact-message" placeholder="Enter text here" maxlength="500" required></textarea>
-          <button id="contact-button" type="submit">Get in touch</button>
-        </form>
+        <div class="contact-description">
+          <h2>I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.</h2>
+        </div>
+        <div class="contact-form">
+          <form action="https://formspree.io/f/moqrnlgd" method="post" id="form">
+            <input type="text" name="name" id="contact-name" placeholder="Full name" maxlength="30" required>
+            <input type="email" name="email" id="contact-email" placeholder="Email address" required>
+            <textarea name="message" id="contact-message" placeholder="Enter text here" maxlength="500" required></textarea>
+            <button id="contact-button" type="submit">Get in touch</button>
+          </form>
+        </div>
+      </section>
+
+      <hr class="footer-hr">
+
+      <section id="footer">
+        <h2>Footer</h2>
+        <footer>
+          <ul>
+            <li><a href="#"><img src="assets/img/GithubIcon.png" alt=""></a></li>
+            <li><a href="#"><img src="assets/img/iconLinkedin.png" alt=""></a></li>
+            <li><a href="#"><img src="assets/img/Vicon.png" alt=""></a></li>
+            <li><a href="#"><img src="assets/img/TwitterIcon.png" alt=""></a></li>
+            <li><a href="#"><img src="assets/img/Micon.png" alt=""></a></li>
+          </ul>
+        </footer>
       </section>
-      <hr>
-      <footer>
-        <ul>
-          <li><a href="#"><img src="assets/img/GithubIcon.png" alt=""></a></li>
-          <li><a href="#"><img src="assets/img/iconLinkedin.png" alt=""></a></li>
-          <li><a href="#"><img src="assets/img/Vicon.png" alt=""></a></li>
-          <li><a href="#"><img src="assets/img/TwitterIcon.png" alt=""></a></li>
-          <li><a href="#"><img src="assets/img/Micon.png" alt=""></a></li>
-        </ul>
```

Comment:
- [ ] Please, kindly add the remaining social media icons, you will need them later.